packages/node/test/integration/uninstall-virtual.spec.ts

```diff
@@ -25,7 +25,7 @@ import {
 } from "./utils";
 
 describe("Node method follows spec - uninstall", () => {
-  jest.setTimeout(30000);
+  jest.setTimeout(80000);
```

Comment:
wtf...does it need to be that high?

Reply:
It does not have to be this high, will revert